sympy/stats/symbolic_multivariate_probability.py

```diff
@@ -0,0 +1,129 @@
+import itertools
+
+from sympy.core.sympify import _sympify
+
+from sympy import MatrixExpr, Expr, ShapeError, ZeroMatrix, Add, Mul, MatMul
+from sympy.stats.rv import RandomSymbol
+from sympy.stats.symbolic_probability import Variance, Covariance, Expectation, is_random
+
+
+class ExpectationMatrix(Expectation, MatrixExpr):
+    """
+    Expectation of a matrix expression.
+    """
+    def __new__(cls, arg, condition=None):
+        arg = _sympify(arg)
+
+        if condition:
+            obj = Expr.__new__(cls, arg, condition)
+        else:
+            obj = Expr.__new__(cls, arg)
+        obj._shape = arg.shape
+        obj._condition = condition
+        return obj
+
+    @property
+    def shape(self):
+        return self._shape
+
+    def doit(self, **hints):
+        expr = self.args[0]
+        condition = self._condition
```

Comment:
Should this be renamed to `expand`, considering analogy with symbolic `Expectation`?
https://github.com/sympy/sympy/blob/b629f57e2db38b519ae4a2f7664efd3add6893ca/sympy/stats/symbolic_probability.py#L152-L161

Reply:
`expand` is meant to be the linear expansion operator.

Actually, `expand` itself is an abuse of terminology. `expand` is supposed to be a polynomial operator to expand multiplications of polynomials into a single polynomial.

Actually, yes. Maybe this method should be renamed to `expand`.